changelogs/current.yaml

```diff
@@ -49,6 +49,10 @@ minor_behavior_changes:
 
 bug_fixes:
 # *Changes expected to improve the state of the world and are unlikely to have negative effects*
+- area: buffer
+  change: |
+    Fixed a bug where the memory will be released if it is coming from ``BufferFragment`` when moving the slices
```

Comment:
Since release notes are user-facing, can you include what configurations or use cases could trigger this? Is it only an issue with out-of-tree extensions?

Reply:
got it. It is an issue with internal listener. I will update it.